README.md

```diff
@@ -9,7 +9,6 @@
 [![LightGraphs](http://pkg.julialang.org/badges/LightGraphs_0.4.svg)](http://pkg.julialang.org/?pkg=LightGraphs&ver=0.4)
 [![LightGraphs](http://pkg.julialang.org/badges/LightGraphs_0.5.svg)](http://pkg.julialang.org/?pkg=LightGraphs)
 
-#### LightGraphs versions in the v0.7 series are designed for Julia 0.5. For Julia 0.4 support, please use the latest version in the 0.6 series.
 
```

Comment:
its kinda ugly and also unnecessary. Most people don't really need to know anything about versioning, they just do Pkg.add, and in any case this information is already reported in the badges and in the Supported Version section of the readme. I can put it back if you want but my opinion is that the readme just looks more professional without that